kedro_mlflow/io/mlflow_dataset.py

```diff
@@ -37,6 +37,7 @@ def _save(self, data: Any):
                 # it will throw exception that file under path already exist.
                 local_path = self._get_save_path()
 
+                print(f"Saving dataset local_path:{local_path}, artifact_path:{self.artifact_path}")
```

Comment:
Print defintely shouldn't be used here. If anything, it should be logged using a logger, but I have doubts whether that's needed at all. Kedro is not logging save paths and I think we shouldn't either.

@Galileo-Galilei what do you think?

Reply:
Yes, I totally agree.